src/components/Form/Form.js

```diff
@@ -51,12 +53,23 @@ export class Form extends Component {
     });
   };
 
+  onTooltipHandler = () => {
+    if (this.state.text.trim() === "" && this.state.title.trim() === "") {
```

Comment:
You should destruct text and title from state, cause you are using this properties several times